database/clickhouse/clickhouse.go

```diff
@@ -9,6 +9,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/ClickHouse/clickhouse-go/v2"
```

Comment:
Can you add a v2 db driver similar to how we use pgxv4 and pgxv5? That way users can decide when to migrate themselves.

Reply:
i don't think it's possible, clickhouse v2 autoregister as "clickhouse" same as v1
while pgx it using different name "pgx5"

Reply:
I've confirmed this behavior in the code:
* https://github.com/ClickHouse/clickhouse-go/blob/v2.34.0/clickhouse_std.go#L122
* https://github.com/ClickHouse/clickhouse-go/blob/v1.5.4/bootstrap.go#L44

Reply:
finally green btw @dhui